documentation/developer-guide/modules/tooling-guide/pages/samm-cli.adoc

```diff
@@ -5,10 +5,16 @@
 
 The SAMM CLI is a command line tool for the validation of Aspect Models and the generation of artifacts, such as documentation or code, from Aspect Models.
 
-TIP: Download latest version: icon:download[] https://github.com/eclipse-esmf/esmf-sdk/releases/download/v{esmf-sdk-version}/samm-cli-{esmf-sdk-version}.jar[samm-cli-{esmf-sdk-version}.jar]
+The SAMM CLI offers multiple deployment options (you can download it bellow):
```

Comment:
The SAMM CLI is available as a native executable for multiple platforms and as an an executable jar: